mesonbuild/compilers/compilers.py

```diff
@@ -137,6 +139,13 @@ def is_library(fname):
                            'release': ['-Os'],
                            'minsize': ['-Oz']}
 
+cuda_buildtype_args = {'plain': [],
+                       'debug': ['-O0', '-g'],
+                       'debugoptimized': ['-O1', '--debug'],
+                       'release': ['-O3', '-Otime'],
+                       'minsize': ['-O3', '-Ospace'],
+                      }
```

Comment:
__[Flake8]__

[E124] closing bracket does not match visual indentation


<!-- Comment by [Sider](https://sider.review) -->